app/controllers/payment-links/get-edit.controller.js

```diff
@@ -33,9 +39,23 @@ module.exports = async function showEditPaymentLink (req, res, next) {
   try {
     const product = await productsClient.product.getByProductExternalId(gatewayAccountId, productExternalId)
     const productCheck = lodash.cloneDeep(product)
-    delete editPaymentLinkData.metadata
+
+    if (!shouldUseInlineReportingColumns) {
```

Comment:
I'd favour reversing this `if` statement to avoid a double negative for the `else` condition. But as this is being removed soon I'm not too fussed.